staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto

```diff
@@ -395,6 +395,10 @@ message ListMeta {
   // identical to the value in the first response, unless you have received this token from an error
   // message.
   optional string continue = 3;
+
+  // the total number of matching keys in the storage. Count equals to
+  // the number of items in the list if Continue is empty.
+  optional int64 count = 4;
```

Comment:
Count might not be the best name for this given that it has a different meaning when continue is set. If we do use 'count', should clearly document what it means when continue is set here too.